src/mscorlib/src/System/Collections/Generic/KeyValuePair.cs

```diff
@@ -62,5 +62,12 @@ public struct KeyValuePair<TKey, TValue> {
             s.Append(']');
             return StringBuilderCache.GetStringAndRelease(s);
         }
+
+        //[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
I think that would be fine for .NET Core but what about .NET Framework? While we could type-forward it down into mscorlib in the past those things have cause a decent amount of fallout. As an example we had a bunch of fallout when we moved ExtensionAttribute down.